options.js

```diff
@@ -118,10 +118,13 @@ function set_file_editorconfig_opts(file, config) {
 module.exports = (doc, type, formattingOptions) => {
 	let base = vscode.workspace.rootPath;
 	let opts = optionsFromVSCode(doc, formattingOptions, type);
-	let configFile;
+	let configFile = vscode.workspace.getConfiguration('beautify').config;
```

Comment:
Settings the `config`ed beautifyrc path at this point makes it the default. Perhaps setting it before the call to find it in the home directory (or even after) is a better choice.

Having a `.beautifyrc` in the same, or a parent directory of the target file should be the default. This allows different formatting options for different directories within a project.

Putting it here would break this functionality once a config path is set.